event/src/Vertex.cxx

```diff
@@ -51,6 +51,45 @@ void Vertex::setPos(const float* pos, bool rotate) {
     }  
 }
 
+void Vertex::setVtxParameters(const TVector3& p1,
+			      const TVector3& p2,
+			      const double m) {
+
+  p1_.SetX(p1.X());
+  p1_.SetY(p1.Y());
+  p1_.SetZ(p1.Z());
+
+  p2_.SetX(p2.X());
+  p2_.SetY(p2.Y());
+  p2_.SetZ(p2.Z());
+  
+  p_ = p1_ + p2_;
+  
+  invM_  = m;
+
+}
+
+
+void Vertex::setVtxParameters(const TLorentzVector& p1,
+			      const TLorentzVector& p2){
+			      
+
+  p1_.SetX(p1.X());
+  p1_.SetY(p1.Y());
+  p1_.SetZ(p1.Z());
+
+  p2_.SetX(p2.X());
+  p2_.SetY(p2.Y());
+  p2_.SetZ(p2.Z());
+  
+  p_ = p1_ + p2_;
+  
+  invM_  = (p1 + p2).M();
+  
```

Comment:
Suggestion is DRY (Don't Repeat Yourself)
```suggestion
  this->setVtxParameters(p1, p2, (p1+p2).M());
```

Reply:
Done.